edx-solutions/xblock-image-explorer
# [MCKIN-8264] Support backwards compatibility view for v1 block

This PR modifies the Image Explorer XBlock to use the old image explorer style, layout and functionality for existing versions while allowing newer versions of the block to use the new layout. 

**Test Instructions**:
1. Install the master branch version of `xblock-image-explorer`.
2. Add a block to a course, and make sure it's working properly. 
3. Install the version of the block from this PR.
4. Check again that the block is working properly and looks and functions the same way as before. 
5. Add another block.
6. Check that this block uses a different, new layout while the old block continues to use the existing layout.

NOTES: 
The v2 version is still in progress, so might have bugs and issues that are out of scope for this change. For this PR it's sufficient to test that newly created blocks use the new layout while older versions of the block use the old layout. 

Also, allowing both versions of the XBlock to work in the same course and page is not a requirement currently, so I have made no effort to allow that. The idea is to use v2 in new courses, and migrate from v1 to v2 one course at a time while maintaining the v1 style so as not to break existing content.
